commit 27582271c88757549cf9e320ca1b754c20de15b5
Author: Richard Gildea <rjgildea@users.noreply.github.com>
Date:   Wed Jan 25 14:52:22 2023 +0000

    Use std::shared_ptr instead of boost equivalent (#837)
    
    Required for dials/dials#2310

diff --git a/xfel/merging/ext.cpp b/xfel/merging/ext.cpp
index d5eaa970b2..bfbdcf6812 100644
--- a/xfel/merging/ext.cpp
+++ b/xfel/merging/ext.cpp
@@ -1,3 +1,4 @@
+#include <memory>
 #include <cctbx/boost_python/flex_fwd.h>
 #include <boost/python/module.hpp>
 #include <boost/python/class.hpp>
@@ -108,7 +109,7 @@ namespace sx_merging {
     const std::map<std::string, size_t> chunk_lookup;
     const std::vector<dials::af::reflection_table> tables;
     dials::af::shared<int> expt_ids;
-    boost::shared_ptr<dials::af::reflection_table::experiment_map_type> expt_map;
+    std::shared_ptr<dials::af::reflection_table::experiment_map_type> expt_map;
 
     experiment_id_splitter_visitor(dials::af::reflection_table &src_,
                                    dials::af::reflection_table::key_type key_,
